packages/gatsby-source-strapi/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "gatsby-source-strapi",
+  "version": "3.0.0",
+  "description": "Gatsby source plugin for building websites using Strapi as a data source",
+  "author": {
+    "email": "hi@strapi.io",
+    "name": "Strapi Solutions",
+    "url": "http://strapi.io"
+  },
+  "maintainers": [
+    {
+      "name": "Strapi Solutions",
+      "email": "hi@strapi.io",
+      "url": "http://strapi.io"
+    }
+  ],
+  "directories": {
+    "lib": "./lib"
+  },
+  "files": [
+    "lib/*",
+    "gatsby-node.js",
+    "index.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/gatsby-uc/gatsby-source-strapi.git"
+  },
+  "bugs": {
+    "url": "https://github.com/gatsby-uc/gatsby-source-strapi/issues"
+  },
+  "license": "MIT",
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "build": "babel src --out-dir lib",
+    "watch": "babel -w src --out-dir lib",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Remove the command if there are no tests. If there are, let's configure this correctly.